packages/odyssey-storybook/src/components/odyssey-ui-shell/UiShell/UiShell.stories.tsx

```diff
@@ -796,6 +797,13 @@ export const WithOdysseyAppContent: StoryObj<UiShellProps> = {
     optionalComponents: {
       ...sharedOptionalComponents,
       banners: <Banner severity="success" text="This is an app!" />,
+      rightSideMenu: (
+        <UserProfile
+          profileIcon={<UserIcon />}
+          orgName="ORG123"
+          userName="test.user@test.com"
+        />
+      ),
```

Comment:
This isn't necessarily what goes here, but it gives us something to look at for now as we don't control what goes into that popup.